yarn.lock

```diff
@@ -128,7 +128,7 @@
     is-wsl "^2.1.1"
     tslib "^2.0.0"
 
-"@oclif/core@^1.0.0", "@oclif/core@^1.0.2":
+"@oclif/core@^1.0.0":
   version "1.0.2"
```

Comment:
This needs to resolve to `1.0.3` too or we won't be rid of the lodash.template dependency.